### DISCUSSION TITLE
Titles not classified?

### DISCUSSION THREAD

**Discussion by annaau-x** [asreview/asreview#1797] [Q&A]
Hi! I have about 3,000 articles. I used the default model. When I run the programme, it does not seem to have classified them because as I export only the relevant titles, I only see the ones from prior knowledge and I reviewed. I really do not know what happened. I hope you can help me. Thanks!

**Comment by J535D165** [comment]
ASReview is not a classification software but a ranking software. This means that you need to screen till you find all your literature of interest. Did you screen records after your prior knowledge selection? 

This video might help to understand the idea of ASReview a bit more: https://www.youtube.com/watch?v=k-a2SCq-LtA&ab_channel=ASReviewTV
